charts/redpanda/values.yaml

```diff
@@ -709,7 +709,13 @@ statefulset:
   sideCars:
     configWatcher:
       enabled: true
-      resources: {}
+      resources:
+        requests:
+          cpu: 1
```

Comment:
An entire vCPU is overkill for a sidecar unless we know for a fact that `configWatch` really needs a full vCPU. I feel like we could probably get away with `100m` here. If there are ramifications for the static CPU scheduler needing whole numbers, we should probably rely on documentation and warnings for that rather than aggressive defaults.

Reply:
I agree with you. I will add comments to describe the static CPU scheduler case.